src/constructs/dynamodb/dynamodb.ts

```diff
@@ -0,0 +1,41 @@
+import { Tags } from "aws-cdk-lib";
+import { Table } from "aws-cdk-lib/aws-dynamodb";
+import type { TableProps } from "aws-cdk-lib/aws-dynamodb";
+import type { GuStack } from "../core";
+
+export interface OptIn {
+  enabled: true;
+}
+
+export interface OptOut {
+  enabled: false;
+  /**
+   * We recommend using DevX Backups where possible. If it is not suitable for your use-case please document
+   * this here so that we can understand why this is switched off when performing security audits.
+   */
+  optOutReason: string;
+}
+
+export interface GuDynamoTableProps extends TableProps {
+  /**
+   * We recommend using DevX Backups to protect your RDS instance's backups.
```

Comment:
These docs need to be updated to mention DynamoDB / use the [DynamoDB link](https://docs.google.com/document/d/11EZtuVHCnjavE9AYLroiuDUsMzbp2ulFBtj4VkyCKNc/edit#heading=h.vwt7syo8ng40).

Reply:
Good catch! Fixed and force-pushed.